public/app/features/admin/ServerStats.tsx

```diff
@@ -80,6 +81,12 @@ export const ServerStats = () => {
               { name: 'Organisations', value: stats.orgs },
               { name: 'Users total', value: stats.users },
               { name: 'Active users in last 30 days', value: stats.activeUsers },
+              ...(config.featureToggles.displayAnonymousStats && stats.activeDevices
+                ? [
+                    { name: 'Active anonymous devices in last 30 days', value: stats.activeDevices },
+                    { name: 'Active anonymous users in last 30 days', value: Math.floor(stats.activeDevices / 3) },
```

Comment:
@eleijonmarck i know this is behind a feature toggle and stuff, but is this intentional? doesn't feel like the number of active users will be `numActiveDevices / 3` 😅

Reply:
@ashharrison90 yea this is intentional. We count the number of users based on devices is activeDevices/3

I can send over the design doc from product. This is products decision

Reply:
...ok 😅

Reply:
@eleijonmarck I can see how it's pretty confusing though. Maybe we can add a comment explaining the thinking (in one of the future PRs)?